cmd/createtree/main.go

```diff
@@ -141,16 +142,28 @@ func newPK(opts *createOpts) (*any.Any, error) {
 	switch opts.privateKeyType {
 	case "PEMKeyFile":
 		if opts.pemKeyPath == "" {
-			return nil, errors.New("empty PEM path")
-		}
-		if opts.pemKeyPass == "" {
-			return nil, fmt.Errorf("empty password for PEM key file %q", opts.pemKeyPath)
+			return nil, errors.New("empty pem_key_path")
 		}
 		pemKey := &keyspb.PEMKeyFile{
 			Path:     opts.pemKeyPath,
 			Password: opts.pemKeyPass,
 		}
 		return ptypes.MarshalAny(pemKey)
+	case "PrivateKey":
+		if opts.pemKeyPath == "" {
+			return nil, errors.New("empty pem_key_path")
+		}
+		pemSigner, err := keys.NewFromPrivatePEMFile(
+			opts.pemKeyPath, opts.pemKeyPass)
```

Comment:
It seems conceptually strange to me to force a file to exist in this case. We could allow for a literal key to be passed via a flag. That possibly means the key gets logged on shell history, unless the user is careful, so there's that as well. I'm not too sure TBH.

WDYT?